kpatch-build/create-diff-object.c

```diff
@@ -3041,11 +3067,11 @@ void kpatch_write_output_elf(struct kpatch_elf *kelf, Elf *elf, char *outfile)
 }
 
 struct arguments {
-	char *args[4];
+	char *args[5];
```

Comment:
Yeah, unfortunately C doesn't have bounds checking and you can't expect a buffer overrun to cause a seg fault.